examples/common/eventdisplay/FairEveMCTracks.cxx

```diff
@@ -149,8 +149,15 @@ InitStatus FairEveMCTracks::Init()
 {
     FairRootManager *mngr = FairRootManager::Instance();
     fContainer = (TClonesArray *)mngr->GetObject("MCTrack");
-    if (fContainer == nullptr)
-        return kFATAL;
+    if (fContainer == nullptr) {
```

Comment:
```suggestion
    if (!fContainer) {
```

Same as above, see [ES.87](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es87-dont-add-redundant--or--to-conditions).